lesson-5/Chen_Homework Source Files/style.css

```diff
@@ -0,0 +1,38 @@
+body {
+  font-family: "Work Sans", sans-serif;
+}
+.container {
+	position:fixed;
+	top:50%;
+	left:50%;
+	transform: translate(-50%,-50%);
+}
+.colorBox{
+	background-color: #f7f2f2;
+	
+	padding:20px;
+	width:235px;
+
+}
+
+a{background-color:white;
+	display:block;
+	text-align:center;
+	border-radius:5px;
+	width:200px;
```

Comment:
Since you've declared `display: block;` here, this `a` will always take up the whole width of its container, so you don't need to give it a specific width (unless you wanted it to be smaller!)

In this case, just giving `colorBox` some padding would take care of all the spacing we need in this section.